internal/schema/resource.go

```diff
@@ -25,6 +26,7 @@ type Resource struct {
 	UsageSchema       []*UsageItem
 	EstimateUsage     EstimateFunc
 	EstimationSummary map[string]bool
+	InfracostMetadata map[string]gjson.Result
```

Comment:
Does this still need to be called `InfracostMetadata` at this point? Should it be called `Metadata` since that maps to the `output.Resource` struct?

Reply:
Yeah agreed